docs/tutorials/gs_continuous_space.ipynb

```diff
@@ -0,0 +1,1210 @@
+{
```

Comment:
<p>to do or to remove?</p><p> </p><p>by the way the tutorial is well done, so if you add something more as well written as the above we would be very happy.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>yes agreed, maybe actually Helium could be a separate tutorial, where we show also the condensate etc, we can ask David to do it later on </p>